src/toil/batchSystems/gridengine.py

```diff
@@ -90,6 +91,8 @@ def getjobexitcode(sgeJobID):
 
         process = subprocess.Popen(args, stdout = subprocess.PIPE,stderr = subprocess.STDOUT)
         for line in process.stdout:
+            logger.debug(args)
```

Comment:
Are these really needed? If so, they need a little more contextual info in the log message.